packaging_ignore.yml

```diff
@@ -21,6 +21,7 @@ base:
   - ".*: warning: 'RSA_generate_key_ex' is deprecated \\[-Wdeprecated-declarations\\]"
   - ".*: warning: 'EVP_PKEY_assign' is deprecated.*"
   - "\\d+ warnings generated."
+  - "configure: WARNING: unrecognized options: --without-pg-version-check"
```

Comment:
We are adding this warning temporarily. @onderkalaci told me to use this flag in the builds. You can see that I added that flag into both deb rule file and rpm spec file to be able to build pg 15.

Reply:
We already have support for this option. If we observe such a warning there is something wrong with our builds. I agree that we should update debian/rules and citus.spec files. However I do not think that we should ever allow configure to throw this warning.

Reply:
I actualy does not understand what you mean. We do not throw an error specifically for this warning. We have a warning detector machanism which throws error other than items in packaging-ignore file. Therefore since this warning appears in the logs because we add this flag into our configuration, without adding this warning into ignore file, warning detector throws error.